README.md

```diff
@@ -24,12 +24,42 @@
 
 This project is split in back-end and front-end: 1). the front-end can be found in **`./frontend-scaffoldeth`** ; 2). the back-end can be found in **`./backend-nestjs/`**.
 
-The idea is to serve the front-end as a static _html_ file using **[GitHub pages](https://andysign.github.io/token-vote-dapp/) and at the same time host the back-end in the cloud, as in, more exactly using the **[back4app.com](https://www.back4app.com/)** free cloud platform ( the back-end can be found at: **[https://tokenvotedapp-9jc31xz7.b4a.run/](https://tokenvotedapp-9jc31xz7.b4a.run/)** ).
+The idea is to serve the front-end as a static _html_ file using **[GitHub pages](https://andysign.github.io/token-vote-dapp/)** and at the same time host the back-end in the cloud, as in, more exactly using the **[back4app.com](https://www.back4app.com/)** free cloud platform ( the back-end can be found at: **[https://tokenvotedapp-9jc31xz7.b4a.run/](https://tokenvotedapp-9jc31xz7.b4a.run/)** ).
 
 For more information about the front-end read the README file from **[./frontend-scaffoldeth](./frontend-scaffoldeth/)**.
 
 Similarly, for more information about the back-end read the README file from **[./backend-nestjs/README.md](./backend-nestjs/README.md)**.
 
-<img width="1321" alt="image" src="https://github.com/andysign/token-vote-dapp/assets/11134288/3f9352a4-ba16-4b6d-9c94-0748790a59f6">
+```
+┌─────────────────────────────────────────────┐
+│                                             │
+│ FRONTEND                                    │
+│                                             │
+│ https://andysign.github.io/token-vote-dapp/ │
+│                                             │
+└─────────────────────────────────────────────┘
+
+  │                                       ▲
+  │                                       │
+  ▼                                       │
+
+┌─────────────────────────────────────────────┐
+│                                             │
+│ BACKEND                                     │
+│                                             │
+│ https://tokenvotedapp-9jc31xz7.b4a.run/     │
+│                                             │
+└─────────────────────────────────────────────┘
+```
+
+High-level Diagram ( generated w/ https://asciiflow.com/ )
+
+This can be also run in dev mode.
```

Comment:
maybe add information how? Or mention it's in the particular READMEs

Reply:
I feel like there is a lot of important one can copy and paste from the Descriptions of different PRs. I guess this should include the minimum with more detail stuff in the two separate READMEs ( the one in the front end and the one in the backend ).